test/com/google/javascript/jscomp/parsing/NewParserTest.java

```diff
@@ -538,10 +532,9 @@ public void testJSDocAttachment5() {
    * propagate to following code due to {@link JSDocInfo} aggregation.
    */
   public void testJSDocAttachment6() throws Exception {
```

Comment:
That test specifically looked to be testing more than simple misplacement of the JSDoc. It seemed to be testing whether misplaced annotations affected later code which is why I left it.